### DISCUSSION TITLE
ROUNDS BEING ENDED, console, chat, and settings get messed up constantly

### DISCUSSION THREAD

**Discussion by girlyguppy** [FakeFishGames/Barotrauma#14994] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Basically, the chat type box clears any time someone joins/switches a team (and maybe other causes too, such as setting changes). 

Also when someone joins, leaves, or respawns, you get clicked out of the f3 console. 
![image](https://github.com/user-attachments/assets/b97cd968-dcb7-4569-9bf4-6e45f96f99a8) (a better description of the issue)

and the worst problem by far, is that ROUNDS GET ENDED when an admin joins the server. For whatever reason, the "win score" gets reset to 1, even mid-round, so if an admin joins mid-round, the round will end after 1 more kill.

https://github.com/user-attachments/assets/8906fbc0-d11e-40fb-ad8e-115fb06578a3 (a video of an admin leaving then rejoining, the score gets reset from 612 to 1. this even happens during active rounds, meaning the round will be ended if the admin has to rejoin or joins late)

### Reproduction steps
CHAT CLEAR ISSUE
-type something in chat but don't send
-join a team, witness the chat message being cleared. happens over and over in crowded lobbies, making text chat impossible.
====================
CONSOLE ISSUE
-uhhh idk use the console, then have people join
-witness you getting clicked out of the console text box
====================
SCORE RESET ISSUE
-I'm assuming admins specifically cause this, so make someone an admin
-set the win score to >1
-have an admin leave and rejoin, witness the score setting being automatically set to 1.
====================

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.18.1 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report(s)!

The first two seem to be duplicates of https://github.com/FakeFishGames/Barotrauma/issues/14981. I unfortunately could not find a way to reproduce the last issue.
